src/deployments/kubernetes/README.md

```diff
@@ -3,52 +3,87 @@
 This folder provides a basic setup of a set of
 [Kubernetes](https://kubernetes.io/) files to setup a cluster.
 
-## Local Development
+## Initial Kubernetes Setup
 
-For local development, you need to use
-[minikube](https://minikube.sigs.k8s.io/docs/start/).
+### Option 1: Docker Desktop
```

Comment:
I realized that I could use Docker Desktop directly instead of minikube. It is working better for me.